README.md

```diff
@@ -2,11 +2,41 @@
 
 ## Getting Started
 
-1. Clone this repository
-2. Install packages with `npm install`
-3. Run with `npx tsx index.ts`
+1. Install npm package: 
 
-## Documentation
+```
+npm install git+https://github.com/irods/irods_client_http_typescript
+```
 
--   To browse the documentation for this library, open "docs/index.html"
--   If updates are made to the code, run "npx typedoc" to update the documentation html
+2. Import `Wrapper` class:
+
+```
+import { Wrapper } from 'irods_client_http_typescript'
+```
+
+3. Create a url configuration object, containing info about the irods http api that's running on your local machine:
+
+```
+const urlComponents: URLComponentsType = {
+    host: '<host>',
+    port: '<port>',
+    version: '<version>',
+}
+```
+
+4. Create a wrapper object, passing in the url components, as well as the username and password you want to log in with:
+```
+const api = new Wrapper(urlComponents, '<username>', '<password>')
+```
+
+5. Authenticate the wrapper object within an asynchronous context:
+
+```
+(async () => {
+    await api.authenticate()
+})
+```
```

Comment:
Yes, and I've tried to not make it required but haven't found any success. It would be ideal if the user could authenticate in a non-async context from a convenience standpoint.

Reply:
Not a problem. We can work on improving this at a later time.